### PR TITLE
Automatic merge from master to 6.0 branch

### PR DESCRIPTION
**Pull request by roncohen**

### CODE REVIEW

File: _beats/dev-tools/packer/version.yml [hunk 1]
```diff
@@ -1 +1 @@
-version: "6.0.0-rc2"
+version: "6.1.0"
```
Comment (ruflin) [inline _beats/dev-tools/packer/version.yml]: I think that will break packaging. I would expect that `update-beats` would overwrite this file and set it to the correct version.

Comment (ruflin) [reply _beats/dev-tools/packer/version.yml]: @roncohen Just saw that you changed the base branch after update beats. What did you rebase this on? 6.0 or 6.x? If 6.x that would explain the above issue.

Comment (roncohen) [reply _beats/dev-tools/packer/version.yml]: This was the command: `./_beats/dev-tools/merge_pr master 6.0`

Comment (ruflin) [reply _beats/dev-tools/packer/version.yml]: Ok, that sounds good and you see afterwards you also updated beats. Need to check what is off here.

**Pull request closed without merging**